bundle/deploy/terraform/pkg.go

```diff
@@ -15,18 +15,35 @@ const TerraformVersionEnv = "DATABRICKS_TF_VERSION"
 const TerraformCliConfigPathEnv = "DATABRICKS_TF_CLI_CONFIG_FILE"
 const TerraformProviderVersionEnv = "DATABRICKS_TF_PROVIDER_VERSION"
 
+// Terraform CLI version to use and the corresponding checksums for it. The
+// checksums are used to verify the integrity of the downloaded binary. Please
+// update the checksums when the terraform version is updated. The checksums
+// were obtained from https://releases.hashicorp.com/terraform/1.5.5.
```

Comment:
Also please note that the hashes are not used here but only colocated to include in the debug output.

Is there some way to confirm their validity? E.g. an integration test to confirm they match the hashes of actually downloaded binaries?

Reply:
added unit test to assert on the checksums wrt the terraform version.